src/Evolution/Systems/GeneralizedHarmonic/ConstraintDamping/TimeDependentTripleGaussian.hpp

```diff
@@ -0,0 +1,220 @@
+ * are scaled by a domain::FunctionsOfTime::FunctionOfTime.
+ *
+ * \details The function \f$f\f$ is given by
+ * \f{align}{
+ * f = C + \sum_{\alpha=1}^3
+ * A_\alpha \exp\left(-\frac{(x-(x_0)_\alpha)^2}{w_\alpha^2(t)}\right).
+ * \f}
+ * Input file options are: `Constant` \f$C\f$, `Amplitude[1-3]`
+ * \f$A_\alpha\f$, `Width[1-3]` \f$w_\alpha\f$, `Center[1-3]
+ * `\f$(x_0)_\alpha\f$, and `FunctionOfTimeForScaling`, a string naming a
+ * domain::FunctionsOfTime::FunctionOfTime in the domain::Tags::FunctionsOfTime
+ * that will be passed to the call operator. The function takes input
+ * coordinates \f$x\f$ of type `tnsr::I<T, VolumeDim, Fr>`, where `T` is e.g.
+ * `double` or `DataVector`, `Fr` is a frame (e.g. `Frame::Inertial`), and
+ * `VolumeDim` is the dimension of the spatial volume. The Gaussian widths
+ * \f$w_\alpha\f$ are scaled by the inverse of the value of a scalar
+ * domain::FunctionsOfTime::FunctionOfTime \f$f(t)\f$ named
+ * `FunctionOfTimeForScaling`: \f$w_\alpha(t) = w_\alpha / f(t)\f$.
+ */
+template <size_t VolumeDim, typename Fr>
+class TimeDependentTripleGaussian : public DampingFunction<VolumeDim, Fr> {
+ public:
+  struct Constant {
+    using type = double;
+    static constexpr Options::String help = {"The constant."};
+  };
+
+  struct Amplitude1 {
+    using type = double;
+    static constexpr Options::String help = {"The amplitudes of Gaussian 1."};
+  };
+
+  struct Width1 {
+    using type = double;
+    static constexpr Options::String help = {
+        "The unscaled widths of Gaussian 1."};
+    static type lower_bound() noexcept { return 0.; }
+  };
+
+  struct Center1 {
+    using type = std::array<double, VolumeDim>;
+    static constexpr Options::String help = {"The centers of Gaussian 1."};
+  };
+
+  struct Amplitude2 {
+    using type = double;
+    static constexpr Options::String help = {"The amplitudes of Gaussian 2."};
+  };
+
+  struct Width2 {
+    using type = double;
+    static constexpr Options::String help = {
+        "The unscaled widths of Gaussian 2."};
+    static type lower_bound() noexcept { return 0.; }
+  };
+
+  struct Center2 {
+    using type = std::array<double, VolumeDim>;
+    static constexpr Options::String help = {"The centers of Gaussian 2."};
+  };
+
+  struct Amplitude3 {
+    using type = double;
+    static constexpr Options::String help = {"The amplitudes of Gaussian 3."};
+  };
+
+  struct Width3 {
+    using type = double;
+    static constexpr Options::String help = {
+        "The unscaled widths of Gaussian 3."};
+    static type lower_bound() noexcept { return 0.; }
+  };
+
+  struct Center3 {
+    using type = std::array<double, VolumeDim>;
+    static constexpr Options::String help = {"The centers of Gaussian 3."};
+  };
+
+  struct FunctionOfTimeForScaling {
+    using type = std::string;
+    static constexpr Options::String help = {"The name of the FunctionOfTime."};
+  };
+
+  using options = tmpl::list<Constant, Amplitude1, Width1, Center1, Amplitude2,
```

Comment:
What do you think about:
```cpp
template <size_t GaussianNumber>
struct GaussianOpts {
  using options = tmpl::list<Amplitude, Width, Center>;
  static constexpr Options::String help = {"Parameters for one of the Gaussians."};
  static std::string name() noexcept {
    return "Gaussian" + std::to_string(GaussianNumber);
  }

  GaussianOpts(const double amplitude_in, const double width_in,
               const std::array<double, VolumeDim> center_in);

  double amplitude;
  double width;
  std::array<double, VolumeDim> center;
};

// In cpp file
GaussianOpts::GaussianOpts(const double amplitude_in, const double width_in,
                           const std::array<double, VolumeDim> center_in)
    : amplitude(amplitude_in), width(width_in), center(center_in) {}
```
and then listing
```cpp
using options = tmpl::list<Constant, GaussianOpts<1>, GaussianOpts<2>, GaussianOpts<3>>
```
This would make the options cluster together a bit more in the input file. Another way to achieve this without the additional class would be option groups:
```cpp
template <size_t GaussianNumber>
struct Gaussian {
  static constexpr Options::String help = {
      "Parameters for one of the Gaussians."};
  static std::string name() noexcept {
    return "Gaussian" + std::to_string(GaussianNumber);
  }
};
```
and then have
```cpp
template <typename Group>
struct Amplitude {
  using group = Group;
  using type = double;
  static constexpr Options::String help = {"The amplitudes of Gaussian."};
};

template <typename Group>
struct Width {
  using group = Group;
  using type = double;
  static constexpr Options::String help = {
      "The unscaled widths of Gaussian."};
  static type lower_bound() noexcept { return 0.; }
};

template <typename Group>
struct Center {
  using group = Group;
  using type = std::array<double, VolumeDim>;
  static constexpr Options::String help = {"The centers of Gaussian."};
};

using options = tmpl::list<Constant, Amplitude<Gaussian<1>>, Wideth<Gaussian<1>>, ...>;
```
Using option groups would mean no changes to the constructors or anything like that, just changing the option classes a bit (I think that's easier and probably even cleaner)